lib/server/routes/buckets.js

```diff
@@ -948,28 +949,66 @@ BucketsRouter.prototype.getFile = function(req, res, next) {
       return next(new errors.NotFoundError('Bucket not found'));
     }
 
-    BucketEntry.findOne({
-      _id: req.params.file,
-      bucket: bucket._id
-    }).populate('frame').exec(function(err, entry) {
+    User.findOne({
```

Comment:
This should be available at `req.user` already since this is an authenticated route, no?